backend/TaskTree/Models/TaskTreeContext.cs

```diff
@@ -71,4 +72,5 @@ private void OnBeforeSave()
     public DbSet<User> Users { get; set; }
     public DbSet<Task> Tasks { get; set; }
     public DbSet<Project> Projects { get; set; }
+    public DbSet<TaskTree.Models.Responses.UserResponse> UserResponse { get; set; }
```

Comment:
this is not needed. the DbSets define data we actually get from the database, UserResponse is just an object that defines the data we return from the json api.